src/main/views/daily-cause-list.njk

```diff
@@ -0,0 +1,107 @@
+        {% set hearingCount = 1 %}
+        {% for session in courtRoom['session'] %}
+          {% set sessionCount = sessionCount + 1 %}
+          {% set sessionChannel = session['sessionChannel'][0] %}
+          <div class="govuk-accordion__section ">
+            <div class="govuk-accordion__section-header">
+              <h2 class="govuk-accordion__section-heading">
+              <span class="govuk-accordion__section-button" id="accordion-default-heading-{{ sessionCount }}">
+                {{ courtroomAccordionHeading }} {{ courtRoom['courtRoomName'] }} {{ session['judiciary'][0]['johTitle'] }} {{ session['judiciary'][0]['johNameSurname'] }}
+              </span>
+              </h2>
+            </div>
+            <div id="accordion-default-content-{{ sessionCount }}" class="govuk-accordion__section-content" aria-labelledby="accordion-default-heading-{{ sessionCount }}">
+
+              <table class="govuk-table">
+                <thead class="govuk-table__head">
+                <tr class="govuk-table__row">
+                  <th scope="col" class="govuk-table__header">Case ID</th>
+                  <th scope="col" class="govuk-table__header">Case name</th>
+                  <th scope="col" class="govuk-table__header">Hearing type</th>
+                  <th scope="col" class="govuk-table__header">Hearing platform</th>
+                  <th scope="col" class="govuk-table__header">Start time</th>
+                  <th scope="col" class="govuk-table__header">Duration</th>
+                </tr>
+                </thead>
+
+                <tbody class="govuk-table__body">
+                {% for sitting in session['sittings'] %}
+                  {% set hearingPlatform = sitting['channel'][0] %}
+                  {% if sitting['channel'][0] === '' %}
+                    {% set hearingPlatform = sessionChannel %}
+                  {% endif %}
+                  {% set duration = sitting['duration'] %}
+                  {% if sitting['duration'] !== '' %}
+                    {% set duration = sitting['duration'] + ' ' + hearingDuration %}
+                  {% endif %}
+                  {% for hearing in sitting['hearing'] %}
+                    {% for case in hearing['case'] %}
+                      {% if courtRoom['totalHearing'] === hearingCount %}
+                        {% set noBorder = 'no-border-bottom' %}
+                      {% endif %}
+                      <tr class="govuk-table__row">
+                        <td class="govuk-table__cell {{ noBorder }}">{{ case['caseNumber'] }}</td>
+                        <td class="govuk-table__cell {{ noBorder }}">{{ case['caseName'] }}</td>
+                        <td class="govuk-table__cell {{ noBorder }}">{{ hearing['hearingType'] }}</td>
+                        <td class="govuk-table__cell {{ noBorder }}">{{ hearingPlatform }}</td>
+                        <td class="govuk-table__cell {{ noBorder }}">{{ sitting['startTime'] }}</td>
```

Comment:
Just stood it up and looks good! Only thing I've noticed in the start time, if before 10 is preceded with a 0 , e.g rather than 9am it's 09am.

Would that be a quick thing to update so that it removes the initial 0?